Jseldess/contributing

jseldess opened the pull request:
Address #14.

@bdarnell, @spencerkimball, @petermattis, @jess-edwards, I've added basic docs on contributing to cockroach and cockroach docs. You can see html versions here:

- [Contribute to CockroachDB](http://www.cockroachlabs.com/docs/contribute-to-cockroachdb.html)
- [Improve the Docs](http://www.cockroachlabs.com/docs/improve-the-docs.html)

Any thoughts?

bdarnell commented on contribute-to-cockroachdb.md:
```diff
@@ -18,4 +18,7 @@ When you see a bug or have improvements to suggest, please [open an issue](https
+For development-related question, there are two easy ways to get in touch:
+
+- [Post to our Developer mailing list](https://groups.google.com/forum/#!forum/cockroach-db). Please join first or your messages may be held back for moderation.
+- [Chat us on Gitter](https://gitter.im/cockroachdb/cockroach)
```
"Chat with us" instead of "Chat us"

bdarnell commented:
LGTM. 

It's good to have a file named `CONTRIBUTING.md` since github will show a link to this file if it exists during the pull request process. It could just be a small file that links out to the page on the main site.

jseldess commented:
Thanks, @bdarnell. I changed "chat us" to "chat with us" and added a CONTRIBUTING.md file that will just point them to the appropriate page in the docs. 

@spencerkimball, @petermattis, @jess-edwards, let me know if you have any other changes.
